charts/matrixone-operator/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "matrixone-operator.name" . }}-cm
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "matrixone-operator.labels" . | nindent 4 }}
+data:
+  defaultArgs: |
```

Comment:
As we discussed offline, the config file name should be more general, `defaultArgs` is one of the purpose of this config file.